pandas/core/tools/datetimes.py

```diff
@@ -538,7 +538,7 @@ def to_datetime(
     infer_datetime_format=False,
     origin="unix",
     cache=True,
-):
+):  # noqa: D207
```

Comment:
same question here?

Reply:
there looks to be a problem with the use of continuation. The next line is flagged as underindented (D207).

Reply:
It's black that requires this, so I would maybe not include the D207 check then, if it gives conflicts with black.

Reply:
disabling D207 on one or two functions is better than disabling on all, surely?

Reply:
see https://github.com/PyCQA/pydocstyle/issues/234